osmosis-1/osmosis.zone_assets.json

```diff
@@ -2579,7 +2579,7 @@
       "base_denom": "aISLM",
       "path": "transfer/channel-1575/aISLM",
       "osmosis_verified": false,
-      "osmosis_unlisted": true
+      "osmosis_validated": true
```

Comment:
Was this token validated on the frontend? or should I keep this on my todo list? I was never able to validate it, which is why it was left as unlisted.
Note: `osmosis_validated` will be taking over this role of keeping new assets hidden until they've been validated, and will default to false if not included, making the listing process less confusing